api/src/Controller/BaseController.php

```diff
@@ -493,7 +493,7 @@ public function getDepartment($id)
                 $select->subselect()
                     ->columns('COUNT(DepartmentID)')
                     ->from('`Departments` d2')
-                    ->whereEquals(['d2.ParentDepartmentID' => 'd1.DepartmentID'])
+                    ->where('d2.ParentDepartmentID = d1.DepartmentID')
```

Comment:
Isn't this the same?  Does anything meaningful change here?

Reply:
When calling `api/department` the value of `child_count` was `null` with the original code and now correctly reflects the `child_count` with the new changes.

So unless `child_count` was supposed to be `null`, I believe there may have been an error in syntax. It could probably be achieved with `whereEquals` but I happened to just run it with `where` and got an accurate count.

Reply:
as below.  it was bad single quotes...